.github/workflows/canary.yml

```diff
@@ -0,0 +1,40 @@
+name: Release
+
+# Releases a canary release when new commits merge to main
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
```

Comment:
```suggestion
```
Don't auto publish canary.

We should include this as a workflow_dispatch because needing to publish something to test is something that devs need to be able to do without interupting the normal release process